optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/resources/i18n/OptaShiftUIConstants.java

```diff
@@ -13,6 +13,9 @@
 	@TranslationKey(defaultValue="AbstractRosterViewPanel.startSolving")
 	final String AbstractRosterViewPanel_startSolving = "Click the <i>Solve</i> button to start solving.";
 	
+	@TranslationKey(defaultValue="AbstractRosterViewPanel.terminateEarly")
+	final String AbstractRosterViewPanel_terminateEarly = "Terminate";
+	
```

Comment:
code style (non-merge stopping): some of the older entries in this file aren't indented properly - please take that along in a future PR somewhere.